lib-messagebus/datacommunicator/kafkacomm.go

```diff
@@ -112,7 +112,7 @@ func KafkaConnect(kp *KafkaPacket, messageQueueConfigPath string) error {
 	// Creation of TLS Config and Dialer
 	tls, e := TLS(mq.KAFKACertFile, mq.KAFKAKeyFile, mq.KAFKACAFile)
 	if e != nil {
-		log.Printf("%v", e)
+		log.Error("%v" + e.Error())
```

Comment:
I think "%v" is a formatting directive. If used with '+', Go will consider it as a string. We should either use it with logrus.Errorf or we should remove it. Please modify this at all the places if the need be.

Reply:
Yes. removed it